tests/sqlparser_hive.rs

```diff
@@ -537,6 +537,33 @@ fn parse_use() {
     );
 }
 
+#[test]
+fn parse_hive_unary_ops() {
+    let hive_unary_ops = &[("!", UnaryOperator::SpecialNot)];
+
+    for (str_op, op) in hive_unary_ops {
```

Comment:
the single element for loop seems superflous, we can write the SQL query directly?

Reply:
Yes, I will change it. :)